lib/penelope/ml/registry.ex

```diff
@@ -20,8 +20,10 @@ defmodule Penelope.ML.Registry do
     ptb_digit_tokenizer: Penelope.ML.Text.PTBDigitTokenizer,
     count_vectorizer: Penelope.ML.Text.CountVectorizer,
     regex_vectorizer: Penelope.ML.Text.RegexVectorizer,
+    pos_featurizer: Penelope.ML.Text.POSFeaturizer,
     token_featurizer: Penelope.ML.Text.TokenFeaturizer,
     word2vec_mean_vectorizer: Penelope.ML.Word2vec.MeanVectorizer,
+    pos_tagger: Penelope.NLP.POSTagger,
```

Comment:
I don't think you'd ever put the tagger in a pipeline, because it doesn't implement any of the transform/predict/predict_sequence conventions. I see the POSTagger as analogous to the IntentClassifier module, which is just a standalone thing that wraps a pipeline. So, it shouldn't be in the registry. The featurizer makes sense, though.